tools/wltests/platforms/xz2_android/build_images

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+SCRIPT_DIR=$(dirname $(realpath -s $0))
+BASE_DIR="$SCRIPT_DIR/../.."
+source "${BASE_DIR}/helpers"
+source "${PLATFORM_PATH}/definitions"
+
+################################################################################
+# Build all images required to test a new kernel
+################################################################################
+
+warn="\
+Update the 'definitions' file if needed, but be aware that the values must \
+be in sync with the userspace currently flashed on the device or it won't boot\
```

Comment:
Nit: maybe we'd like to be able set those from command line as options of the lisa-wltest-series command ? Updating the file is a bit of a pain since it's git-versioned. But I guess we can also export the env variable directly so ...

Reply:
I actually tested this by exporting the env variables and it works well enough I think ¯\_(ツ)_/¯

Reply:
Fair enough, I'm happy to keep it that way and not clutter the command line tool with even more parameters than it already has